cmd/c2w/main.go

```diff
@@ -66,6 +66,10 @@ func main() {
 			Name:  "legacy",
 			Usage: "Use \"docker build\" instead of buildx (no support for assets flag)",
 		},
+		cli.BoolFlag{
+			Name:  "net",
+			Usage: "Build with networking capabilities (default: false)",
+		},
```

Comment:
This seems to rely on wamr-specific codes so I think we should enable this with the flag`-net=wamr`.

Reply:
Will try to tackle this, keep in mind I'm not much of a Go developer.